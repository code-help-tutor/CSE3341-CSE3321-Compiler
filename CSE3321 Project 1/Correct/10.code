if (x == y) then in x; else out x;