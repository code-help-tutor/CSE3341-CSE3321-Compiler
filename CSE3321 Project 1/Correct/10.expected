IF
LPAREN
ID[x]
EQUAL
EQUAL
ID[y]
RPAREN
THEN
IN
ID[x]
SEMICOLON
ELSE
OUT
ID[x]
SEMICOLON
